packages/components/src/checkbox-control/style.scss

```diff
@@ -55,7 +57,7 @@
 .components-checkbox-control__input-container {
 	position: relative;
 	display: inline-block;
-	margin-right: $grid-unit-15;
+	margin-right: var(--checkbox-input-margin);
```

Comment:
This change wasn't reflected in the help text margin, so I extracted it into a CSS var (f050242).